config/cable.yml

```diff
@@ -0,0 +1,9 @@
+development:
```

Comment:
We haven't checked in a sample `database.yml` config so I would follow that precedence and do the same with the example action cable config.